lib/executor/variable_looping_vus.go

```diff
@@ -563,10 +563,12 @@ func (vlv VariableLoopingVUs) Run(ctx context.Context, out chan<- stats.SampleCo
 		if newScheduledVUs > currentScheduledVUs {
 			for vuNum := currentScheduledVUs; vuNum < newScheduledVUs; vuNum++ {
 				vuHandles[vuNum].start()
+				vlv.executionState.ModCurrentlyActiveVUsCount(+1)
```

Comment:
See my comment in 5ab0d34b about these changes.

I think this needs a larger refactor, but I don't want to get into it if it's not necessary. Let me know.

Reply:
Yeah, we'll need a simplification/cleanup/refactoring pass eventually, these are getting way too complex for my taste :disappointed: